application_sandbox/timeline_sempahore_cross_queue/main.cpp

```diff
@@ -66,7 +69,7 @@ int main_entry(const entry::EntryData* data) {
       VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME
   }, {
       VK_KHR_TIMELINE_SEMAPHORE_EXTENSION_NAME
-  }, {}, 131072, 131072, 131072, 131072, false, false,  false, 0, false, false, VK_COLORSPACE_SRGB_NONLINEAR_KHR, 
+  }, {}, 131072, 131072, 131072, 131072, true, false,  false, 0, false, false, VK_COLORSPACE_SRGB_NONLINEAR_KHR,
```

Comment:
No blocking this CL, but the parameters to VulkanApplication are pretty much unreadable.

Reply:
I absolutely agree. Most other samples, this is hidden behind a SampleOptions() object that wraps all of this. But when we have to leave the normal sample framework it breaks down.

I think this is up for a refactor some time in the future. (FYI: that false/true is turning on the async compute queue)